bin/tag-and-deploy-docker-image

```diff
@@ -1,13 +1,21 @@
 #!/usr/bin/env bash
 
-# You need to set the following variables in the Travis-GUI:
+# You MUST set the following variables in the Travis-GUI:
 # DOCKER_USERNAME:                 The username on Dockerhub
 # DOCKER_PASSWORD:                 The password on Dockerhub
 #
-# You need to set the following variables (here or the main script):
+# You MUST set the following variables (here or in the main script):
 # REMOTE_IMAGE_NAME:               Docker image name on Dockerhub
 # LOCAL_IMAGE_NAME_AND_TAG:        Local docker image name used for as base for tagging
 # RELEASE_SERVER_SERVICE_HANDLE:   Service handle of livingdocs/release-server
+#
+# You COULD set the following variables (here or in the main script)
+# Based on the set variables the script decides how to tag images
+# If you use Travis, you don't have to set this variables
+# GIT_BRANCH:                      Name of the branch
+# PULL_REQUEST_BRANCH              Pull request number
```

Comment:
yes thx, I will rename the variables anyway. At the moment I try to have a script which is working :-)